### DISCUSSION TITLE
Is Netflix using V3 in production?

### DISCUSSION THREAD

**Discussion by james-deee** [Netflix/conductor#2493] [Q&A] [answered]
Hello, we are considering moving over to V3 of Conductor, but had some questions

1. Is V3 currently being used by Netflix internally in production?
2. Is there a migration guide from V2 to V3? 
3. Is there a Document that describes all of the new things that V3 is offering over V2? (e.g. new UI, what performance improvements, etc). 
4. Are there things that existed in V2 that will not exist in V3?

Thanks, I just haven't had a lot of luck finding answers to the above.

-Jamie

**Comment by peterlau** [comment, marked as answer]
Q1. Yes, both the server & new UI are deployed for production workflows at Netflix.

Q2-4: There is not one single resource for the migration at the moment, but the relevant information can be found between the `CHANGELOG.md` and Release Announcements on the Discussion forum.

**Comment by xiaowei582648206** [comment]
hi bro, did you get a satisfied answer?  I aslo need some help with upgrading to conductor V3.x.x.
